Dynatrace/dynatrace-operator
# Updates schema for Google Cloud Marketplace

# Description

Updates the schema for the Google Cloud Marketplace to reflect the latest changes to the values.yaml and Helm charts.

## How can this be tested?

* Install `mpdev`, see [google documentation for more information](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/tool-prerequisites.md)
* Create an empty GKE cluster
* Apply Googles Application CRD, see [google documentation for more information](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/tool-prerequisites.md)
* Run `hack/gcr/deployer-image.sh' to build and push a new deployer image containing the helm charts
* Run `hack/gcr/deploy.sh` to deploy the deployer image
* In the Google Cloud Console, go to Kubernetes Clusters / Applications, select your cluster and check the deployment was successful


## Checklist
~~- [ ] Unit tests have been updated/added~~
